### PR TITLE
[net9.0] Enable tizen

### LINKED ISSUE #19429: [net9.0] Re-enable Tizen Support on net9 Branch
### Description

Currently, the Tizen support has been temporarily[ removed from the net9 branch](https://github.com/dotnet/maui/pull/19236) of the dotnet/maui repository to ensure the branch builds successfully in the absence of packages from the Samsung team. This issue is created to track the progress of re-enabling Tizen support on the net9 branch once the necessary packages become available.

Work related with #19254

### Steps to Reproduce

Clone the dotnet/maui repository.
Checkout the net9 branch.
Attempt to build the project with Tizen support.

### Link to public reproduction project repository

_No response_

### Version with bug

8.0.3

### Is this a regression from previous behavior?

Not sure, did not test other versions

### Last version that worked well

Unknown/Other

### Affected platforms

Other (Tizen, Linux, etc. not supported by Microsoft directly)

### Affected platform versions

_No response_

### Did you find any workaround?

_No response_

### Relevant log output

_No response_

### PR DESCRIPTION
**Pull request by rmarinho**
### Description of Change

Enables Tizen on the net9 branch now that we have packages.

### Issues Fixed

Fixes #19429

### CODE REVIEW

**Review by jonathanpeppers — COMMENTED** [review]
You probably need to make sure the package is on the `dotnet-public` feed:
```
error NU1101: Unable to find package Samsung.NET.Sdk.Tizen.Manifest-9.0.100-alpha.1. No packages exist with this id in source(s)
```
You can run the pipeline mentioned here:
* https://github.com/dotnet/arcade/blob/main/Documentation/MirroringPackages.md

**Comment by rmarinho** [comment]
Yeah I have mirror yesterday but it failed because of azure issues. I have re run, they should be there now.

**Review by jonathanpeppers — APPROVED** [review]
I don't know how green your CI is after the holiday (if the failures are OK or not).

But the actual code changes LGTM. 👍

File: src/Templates/src/templates/maui-mobile/Platforms/Tizen/tizen-manifest.xml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
-<manifest package="maui-application-id-placeholder" version="0.0.0" api-version="8" xmlns="http://tizen.org/ns/packages">
+<manifest package="maui-application-id-placeholder" version="0.0.0" api-version="9" xmlns="http://tizen.org/ns/packages">
```
Comment (rmarinho) [inline src/Templates/src/templates/maui-mobile/Platforms/Tizen/tizen-manifest.xml:2]: When testing templates.. tizen-manifest file is 7.

Comment (rmarinho) [reply src/Templates/src/templates/maui-mobile/Platforms/Tizen/tizen-manifest.xml:2]:
```
D:\a\_work\1\s\bin\dotnet\packs\Samsung.Tizen.Sdk\8.0.131\targets\Samsung.Tizen.Sdk.Packaging.targets(161,5): error : The api-version specified in tizen-manifest file is 7. [D:\a\_work\_temp\test-intg\Buildmauinet80ReleaseFalse\Buildmauinet80ReleaseFalse.csproj::TargetFramework=net8.0-tizen]
D:\a\_work\1\s\bin\dotnet\packs\Samsung.Tizen.Sdk\8.0.131\targets\Samsung.Tizen.Sdk.Packaging.targets(161,5): error : Current target framework net8.0-tizen is not supported in this api-version. [D:\a\_work\_temp\test-intg\Buildmauinet80ReleaseFalse\Buildmauinet80ReleaseFalse.csproj::TargetFramework=net8.0-tizen]
    0 Warning(s)
    2 Error(s)
```

Comment (rmarinho) [reply src/Templates/src/templates/maui-mobile/Platforms/Tizen/tizen-manifest.xml:2]: Ok the issue is 8.0.3 templates have api-version=7 we need to fix that on sr2

**Comment by JoonghyunCho** [comment]
Looks like Tizen blocking issue is fixed?

**Pull request merged**